### DISCUSSION TITLE
刑具文化 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#437] [General]
# 刑具文化 - Miao Yu | 于淼

https://yufree.cn/cn/2023/10/16/torture-culture/

**Comment by earfanfan** [comment]
史丹利文明此次对于刑具文化的观察过于潦草，简直是在浪费资源，不予批准。哈哈，另外，建议补充对更多刑具种类的观察，比如思想、食物等等。
